src/ansys_sphinx_theme/extension/linkcode.py

```diff
@@ -0,0 +1,220 @@
+"""Docstring missing."""
+import inspect
+import os
+import os.path as op
+import sys
+
+from docutils import nodes
+from docutils.nodes import Node
+import sphinx
+from sphinx import addnodes
+from sphinx.application import Sphinx
+from sphinx.locale import _
```

Comment:
it is used in sphinx localizing and translation in Sphinx documentation. so by using this, in name of source we can locate the source file.

Reply:
I am not understanding this. I thought Sphinx was aware about the extensions because we had to register those in the `app` object. For me, the `_` symbol is related to an unused variable.